src/main/java/com/backbase/ct/bbfuel/data/PaymentsDataGenerator.java

```diff
@@ -61,6 +67,10 @@ public static InitiatePaymentOrder generateInitiatePaymentOrder(String debtorArr
         if (PAYMENT_TYPE_SEPA_CREDIT_TRANSFER.equals(paymentType)) {
             currency.setCurrencyCode("EUR");
             identification = generateIbanIdentification();
+        } else if (PAYMENT_TYPE_US_FOREIGN_WIRE.equals(paymentType)) {
```

Comment:
if you add the bic to the `generateCreditorBank()` you can remove this `else if` branch

Reply:
done